.github/ISSUE_TEMPLATE/general-extension.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report - General Extension
+description: Create a bug report for downloading, installing, or building the extension.
+body:
+- type: markdown
+  attributes:
+    value: |
+      ### Is there an existing issue for this?
+      Please search our [existing issues](https://github.com/microsoft/vscode-cpptools/issues) to see if an issue already exists for the bug you encountered.
+- type: textarea

```

Comment:
Wouldn't this be better as multiple `input` instead of a `textarea`?

Reply:
Thought about making these as individual `input`, but it will make the form look longer than it is and cumbersome. 

Example, there will be multiple large headings per each input. Whereas it would be easier to type and read all the environment info from `textarea`.
![image](https://user-images.githubusercontent.com/38734287/163057442-fce74d62-93eb-4114-88d5-a76f5469e45d.png)
(image is from https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

Reply:
Why are the other templates not also updated? Was that so we could get feedback first?